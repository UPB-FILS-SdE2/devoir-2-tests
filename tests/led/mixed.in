led on 0 0
led brightness 0 0
led off 0 0
led brightness 0 0
led brightness set 4 1 2
led brightness 4 4
led brightness 1 2
led blink 100 5 1 3
led brightness 1 3
quit