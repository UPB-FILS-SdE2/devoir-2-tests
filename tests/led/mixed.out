cmd: Set pixel 0 0 to 9
cmd: 9
cmd: Set pixel 0 0 to 0
cmd: 0
cmd: Set pixel 1 2 to 4
4
cmd: 0
cmd: 4
cmd: Set pixel 1 3 to 9
Sleep for 100 ms
Set pixel 1 3 to 0
Sleep for 100 ms
Set pixel 1 3 to 9
Sleep for 100 ms
Set pixel 1 3 to 0
Sleep for 100 ms
Set pixel 1 3 to 9
Sleep for 100 ms
Set pixel 1 3 to 0
Sleep for 100 ms
Set pixel 1 3 to 9
Sleep for 100 ms
Set pixel 1 3 to 0
Sleep for 100 ms
Set pixel 1 3 to 9
Sleep for 100 ms
Set pixel 1 3 to 0
Sleep for 100 ms
cmd: 0
cmd: 